bindings/node/test/autoEncrypter.test.js

```diff
@@ -110,6 +110,92 @@ describe('AutoEncrypter', function () {
     sandbox.restore();
   });
 
+  describe('#constructor', function () {
+    // This case is driver 4.9.0 and higher.
```

Comment:
optional - we could move the comment into the test description so it's available in test output and won't get out of sync with the tests

```typescript
context('when mongodb exports BSON (driver >= 4.9.0)', ... )
```

Reply:
this would apply to the other context block in this file too